jarbas/dashboard/admin.py

```diff
@@ -88,6 +88,19 @@ def queryset(self, request, queryset):
         return queryset.suspicions() if self.value() == 'yes' else queryset
 
 
+class HasReceiptFilter(JarbasListFilter):
+
+    title = 'nota fiscal digitalizada'
+    parameter_name = 'has_receipt'
+    options = (
+        ('yes', 'Sim'),
+        ('no', 'Não'),
+    )
+
+    def queryset(self, request, queryset):
+        return queryset.has_receipt_url(self.value() == 'yes') if self.value() else queryset
```

Comment:
Shouldn't this be something following this logic?

* If filter is `yes` we show only reimbursements **with** `receipt_url`, i.e. `self.queryset.has_receipt(True)`
* If filter is `no` we show only reimbursements **withot** `receipt_url`, i.e. `self.queryset.has_receipt(False)`
* If there's no filter (or something else than `yes` or `no`) we show everything, i.s. `self.queryset`

It's a three possibilities switch but your code offers just two. [This comment elsewhere](https://github.com/datasciencebr/jarbas/pull/257#discussion_r142478743) might be useful ; )

Reply:
This is indeed a better way!